synapse/handlers/room_batch.py

```diff
@@ -213,9 +213,6 @@ async def persist_state_events_at_start(
                     room_id=room_id,
                     action=membership,
                     content=event_dict["content"],
-                    # Mark as an outlier to disconnect it from the normal DAG
```

Comment:
there's some text in the docstring for this function:

> ... are marked as outliers which disconnects them from the normal DAG.

which sounds like it is no longer correct?

Reply:
Good call, updated the spots that reference `outlier`